test/json_product_generation_test.dart

```diff
@@ -20,7 +20,7 @@ void main() {
     String barcode = '0030000010204';
     ProductQueryConfiguration configurations = ProductQueryConfiguration(
         barcode,
-        language: OpenFoodFactsLanguage.ENGLISH,
+        languages: [OpenFoodFactsLanguage.ENGLISH],
```

Comment:
Too bad none of your tests include at least 2 languages.

Reply:
But... but there's a new test with 3 languages: `multiple translated fields` in `test/api_getProduct_test.dart`

Reply:
Fair enough. It didn't see it at once: so many changes in that file ("language" into "languages") that github didn't want to display the file unless you ask politely!